### DISCUSSION TITLE
Hero 7 USB Power Trigger

### DISCUSSION THREAD

**Discussion by zerotco** [gopro/labs#135] [Feature Requests]
Please help to make  HERO7 to fully support USB Power Trigger

Currently: compatibility: Labs enabled HERO7 (limited)

with command: !uN!S!uE!R HERO7 stops recording when USB power disconnected,
but camera is not properly starts once USB power connected back (just display has a backlight, nothing else,).

**Comment by dnewman-gpsw** [comment]
When adding Labs to HERO7, this was a problem discovered at the time, with no easy solution found.  They are no future Labs updates planned for HERO7.
